src/repr/src/optimize.rs

```diff
@@ -91,11 +91,24 @@ macro_rules! optimizer_feature_flags {
 }
 
 optimizer_feature_flags!({
+    // Enable fast path optimization.
+    enable_fast_path: bool,
+    // Enable consolidation of unions that happen immediately after negate.
+    //
+    // The refinement happens in the LIR ⇒ LIR phase.
     enable_consolidate_after_union_negate: bool,
+    // An exclusive upper bound on the number of results we may return from a
+    // Persist fast-path peek. Required by the `create_fast_path_plan` call in
+    // `peek::Optimizer`.
     persist_fast_path_limit: usize,
```

Comment:
Nit: This isn't really a feature flag but a config. We might want to rename "optimizer features" to "optimizer config" everywhere, but I can see how this would cause a lot of churn, especially because the name `OptimizerConfig` is already used for another type. So feel free to not act on this, it's just something I noticed.

Reply:
I agree. Ideally I would try to have flatten `OptimizerConfig` and `OptimizerFeatures` into one struct generated by a macro, but I didn't want to make the PR too big.